editor/editor_inspector.cpp

```diff
@@ -1959,11 +1959,11 @@ void EditorInspector::_edit_request_change(Object *p_object, const String &p_pro
 	if (changing)
 		return;
 
-	if (p_property == String())
-		update_tree_pending = true;
-	else {
+	if (p_property != String()) {
 		pending.insert(p_property);
 	}
+
+	update_tree_pending = true;
```

Comment:
Stupid Question alert! 😉 
does the `String()` automatically check whether the string is null or empty? 
and also do you think it will be required?  please do tell me if I do not understand something I should still learning here.